### DISCUSSION TITLE
delete old files on onedrive

### DISCUSSION THREAD

**Discussion by bellessecretsshow** [abraunegg/onedrive#2142] [Q&A] [answered]
Hi   no issues with one drive  its set to upload  the files from my proxmox back ups   i just cannot get it to delete the old backups   that are no longer on the samba share. 

Any Ideas?

its running in a debian 11 lxc  container   unprivilaged as i dont need it to download the files again  i just want them sent to the cloud and then removed when they no longer match the onedrive folder on the network that proxmox writes to. 

Thanks

**Comment by abraunegg** [comment, marked as answer]
@bellessecretsshow 
What version of onedrive are you using?

I am trying to understand your use case. By default files are deleted locally and remotely when delete operations occur - however if you use  `--resync` then all of this sync state is lost and any files that are to be removed is lost.

When using `--upload-only` - this will delete files online unless you use the `--no-remote-delete` option.

When using `--download-only` - this will not delete local unless you use the `--cleanup-local-files` option.

How are you using the application?

**Comment by bellessecretsshow** [comment]
i re did it   wiped onedrive and used the monitor on and its working as intended now.  23mb of ram   woooo   

i was running resync   so it was trying to pull down the one drive files to  a read only folder  as its  a samba mount in a lxc container.   so i know have 6tb fully automated backup system and Nas.

**Reply by abraunegg** [reply]
@bellessecretsshow 
Thats great :)

Please remember though - do not use --resync unless the application needs it ..
